app/src/main/kotlin/org/cru/godtools/ui/dashboard/lessons/LessonsLayout.kt

```diff
@@ -29,19 +31,27 @@ internal sealed interface DashboardLessonsEvent {
 @Composable
 @OptIn(ExperimentalFoundationApi::class)
 internal fun LessonsLayout(viewModel: LessonsViewModel = viewModel(), onEvent: (DashboardLessonsEvent) -> Unit = {}) {
-    val lessons by viewModel.lessons.collectAsState()
+    val lessons by viewModel.lessons.collectAsState(emptyList())
+    val selectedLanguage by viewModel.selectedLanguage.collectAsState(Language(Locale.getDefault()))
+
     LazyColumn(contentPadding = PaddingValues(16.dp)) {
-        item("header", "header") { LessonsHeader() }
+        item("header", "header") {
+            LessonsHeader()
+            HorizontalDivider(modifier = Modifier.padding(vertical = 12.dp))
+            LessonFilters()
+        }
 
         items(lessons, { it }, { "lesson" }) { lesson ->
             LessonToolCard(
                 lesson,
+                selectedLanguage = selectedLanguage,
                 onEvent = {
                     when (it) {
                         is ToolCardEvent.OpenTool, is ToolCardEvent.Click -> {
                             viewModel.recordOpenLessonInAnalytics(it.tool)
-                            onEvent(DashboardLessonsEvent.OpenLesson(it.tool, it.lang1))
+                            onEvent(DashboardLessonsEvent.OpenLesson(it.tool, selectedLanguage.code))
```

Comment:
this should be launching in the language specified by the LessonToolCard, which now will be the selectedLanguage